src/main/java/com/any/finishcoursetwo/serviceimpl/ExaminerServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package com.any.finishcoursetwo.serviceimpl;
+
+import com.any.finishcoursetwo.exception.TooLargeAmountQuestionException;
+import com.any.finishcoursetwo.model.Question;
+import com.any.finishcoursetwo.service.QuestionService;
+import org.springframework.stereotype.Service;
+import com.any.finishcoursetwo.service.ExaminerService;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+@Service
+public class ExaminerServiceImpl implements ExaminerService {
+    private final QuestionService questionServiceExaminer;
```

Comment:
```suggestion
    private final QuestionService questionService;
```
поскольку Examiner это данный сервис